packages/golden_toolkit/lib/src/configuration.dart

```diff
@@ -87,12 +87,13 @@ class GoldenToolkitConfiguration {
   /// [deviceFileNameFactory] a func used to decide the final filename for multiScreenGolden() invocations
   ///
   /// [primeAssets] a func that is used to ensure that all images have been decoded before trying to render
-  const GoldenToolkitConfiguration({
```

Comment:
the assertion forced me to remove the const. This is technically breaking, but the assertion feels important.